packages/react/src/radio/Radio.js

```diff
@@ -2,33 +2,40 @@ import { useMergeRefs } from '@tonic-ui/react-hooks';
 import { callAll, isNullish } from '@tonic-ui/utils';
 import React, { forwardRef, useRef } from 'react';
 import { Box } from '../box';
+import { useDefaultProps } from '../default-props';
 import { VisuallyHidden } from '../visually-hidden';
 import { defaultSize, defaultVariantColor } from './constants';
 import RadioControlBox from './RadioControlBox';
 import useRadioGroup from './useRadioGroup';
 import { useRadioStyle } from './styles';
 
-const Radio = forwardRef((
-  {
-    checked,
+const Radio = forwardRef((inProps, ref) => {
+  const {
+    checked: checkedProp,
     children,
     defaultChecked,
-    disabled,
+    disabled: disabledProp,
     id,
     inputProps,
     inputRef: inputRefProp,
-    name,
+    name: nameProp,
     onBlur,
-    onChange,
+    onChange: onChangeProp,
     onClick,
     onFocus,
-    size,
+    size: sizeProp,
     value,
-    variantColor,
+    variantColor: variantColorProp,
     ...rest
-  },
-  ref,
-) => {
+  } = useDefaultProps({ props: inProps, name: 'Radio' });
+
+  let checked = checkedProp;
+  let disabled = disabledProp;
+  let name = nameProp;
+  let onChange = onChangeProp;
+  let size = sizeProp;
+  let variantColor = variantColorProp;
+
   const inputRef = useRef();
   const combinedInputRef = useMergeRefs(inputRefProp, inputRef);
   const styleProps = useRadioStyle({ disabled });
```

Comment:
Please move `const styleProps = useRadioStyle({ disabled });` to line 71.

Reply:
resolved in b7878874bbdbe4cf56972f3aa322f3f14b3cbb3a